SimG4CMS/Forward/interface/BscG4Hit.h

```diff
@@ -105,10 +102,10 @@ class BscG4Hit : public G4VHit {
     G4ThreeVector entry;             //Entry point
     G4ThreeVector entrylp;    //Entry  local point
     G4ThreeVector exitlp;    //Exit  local point
-  double       elem;              //EnergyDeposit of EM particles
-  double       hadr;              //EnergyDeposit of HD particles
-  double       theIncidentEnergy; //Energy of the primary particle
-  G4int          theTrackID;        //Identification number of the primary
+  float       elem;              //EnergyDeposit of EM particles
```

Comment:
is the move from double to float in this class supposed to leave the result unchanged? A priori I would imagine no. This class is used in the BHM and in the FastTimer/MTD sensitive detectors. The latter are PhaseII detectors for which I doubt we have histograms that could spot differences. Not a big dal but...

Reply:
@fabiocos , in the SD classes these values are float or int, so the same type should be members of this hit.